cupyx/jit/_builtin_funcs.py

```diff
@@ -111,6 +151,9 @@ def call(self, env, array, index, value):
 
 builtin_functions_dict = {
     range: RangeFunc(),
+    len: Len(),
+    min: Min(),
+    max: Max(),
```

Comment:
Nitpick: Can we follow the naming convention above and append `Func` for Python built-in functions? So they would read `LenFunc`, `MinFunc`, and `MaxFunc`, respectively.